libbulletc/src/libbulletc/btMotionState_wrap.cpp

```diff
@@ -2,6 +2,8 @@
 
 #include "conversion.h"
 #include "btMotionState_wrap.h"
+// https://stackoverflow.com/questions/3156778/no-matching-function-for-call-to-operator-new
+#include <new>
```

Comment:
Fix compilation error for Android: `No matching function for call to operator new`